api/namex/resources/name_requests/report_resource.py

```diff
@@ -274,16 +274,21 @@ def _get_next_action_text(entity_type_cd: str):
                           'businesses-incorporated-companies/forms-corporate-registry">visit our Forms page</a> to'
                           ' download and submit a form'
             },
-            'CP': 'BC Cooperative Association',
+            'CP': {
+               'DEFAULT': 'To complete your filing, visit <a href="www.bcregistry.ca/business/auth/">www.bcregistry.gov.bc.ca</a>'
+                          ' for more information'
+            },
             'BC': {
-               'DEFAULT': 'To complete your filing, visit <a href="bcregistry.ca/business">www.bcregistry.gov.bc.ca</a>'
+               'DEFAULT': 'To complete your filing, visit <a href="www.bcregistry.ca/business/auth/">www.bcregistry.gov.bc.ca</a>'
```

Comment:
BC types aren't in lear yet, isn't this change too early?

Reply:
the previous one "bcregistry.ca/business" does not work, instead, the link "www.bcregistry.ca/business/auth" works.